superset/assets/src/visualizations/nvd3/utils.js

```diff
@@ -37,6 +37,20 @@ export function drawBarValues(svg, data, stacked, axisFormat) {
     });
 }
 
+// Formats the series key to account for a possible NULL value
+function getFormattedKey(seriesKey, toDompurify) {
+    let key = '';
+    if (toDompurify === true) {
+        key = dompurify.sanitize(seriesKey);
+    } else {
+        key = seriesKey;
+    }
+    if (key === '') {
```

Comment:
are you assuming `null` here? can you explicitly check for `null` and operate on that logic instead of an empty string?